check_tags.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-GITHUB_RESPONSE=$(curl -s -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/vscodium/vscodium/releases/tags/$LATEST_MS_TAG)
+GITHUB_RESPONSE=$(curl -s -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/435vic/vscodium/releases/tags/$LATEST_MS_TAG)
```

Comment:
Can you change this back to `vscodium`?

Reply:
Oh, sorry, commited on the wrong branch. Fixing it now.